e2e/utils/DriverHelper.ts

```diff
@@ -445,10 +446,15 @@ export class DriverHelper {
         }, timeout);
     }
 
+
+
     async switchToSecondWindow(mainWindowHandle: string) {
         await this.waitOpenningSecondWindow();
         const handles: string[] = await this.driver.getAllWindowHandles();
         handles.splice(handles.indexOf(mainWindowHandle), 1);
         await this.driver.switchTo().window(handles[0]);
     }
+async performRightClickOnActiveElement() {
+    await this.driver.actions().click(this.driver.switchTo().activeElement(), Button.RIGHT).perform();
```

Comment:
It's hard to read this chained commands.
Could you extract `this.driver.switchTo().activeElement()` as variable to make it more readable?

Reply:
removed (unnecessary)